staging/src/k8s.io/code-generator/cmd/validation-gen/validation.go

```diff
@@ -362,6 +375,14 @@ func builtinTagDocs() []validators.TagDoc {
 			Description: "<field-name>",
 			Docs:        "This values names a field of a list's value type.",
 		}},
+	}, {
+		Tag:         subfieldTag,
+		Description: "Declares a validation for a specified subfield of the struct. The subfield must be a direct field of the struct, or of an embedded struct",
+		Contexts:    []validators.TagContext{validators.TagContextField},
+		Payloads: []validators.TagPayloadDoc{{
+			Description: "<subfield-json-name>=<validation-tag>=<args>",
```

Comment:
This description isn't right?  If the tag is `+k8s:subfield(<field>)` the payload is `<validation-tag>` -- any other validation tag.

Reply:
Ah good catch thanks.  This is fixed now